.storybook/preview.js

```diff
@@ -66,23 +61,9 @@ export const globalTypes = {
 const themeDecorator = (story, { globals }) => {
   const { theme } = globals;
 
-  switch (theme) {
-    case THEMES.NG_LIGHT:
-      styleElement.textContent = legacyUiStyles;
-      document.body.removeAttribute(cdsThemeAttribute);
-      document.body.style.backgroundColor = getClrUiAppBackgroundColor(theme);
-      break;
-    case THEMES.NG_DARK:
-      styleElement.textContent = legacyUiDarkStyles;
-      document.body.removeAttribute(cdsThemeAttribute);
-      document.body.style.backgroundColor = getClrUiAppBackgroundColor(theme);
-      break;
-    default:
-      styleElement.textContent = `${cdsCoreAndShimStyles.join('')}`;
-      document.body.setAttribute(cdsThemeAttribute, theme === THEMES.CORE_LIGHT ? 'light' : THEMES.CORE_DARK);
-      document.body.style.backgroundColor = null;
-      break;
-  }
+  styleElement.textContent = `${cdsCoreAndShimStyles.join('')}`;
+  document.body.setAttribute(cdsThemeAttribute, theme === THEMES.CORE_LIGHT ? 'light' : THEMES.CORE_DARK);
```

Comment:
```suggestion
  document.body.setAttribute(cdsThemeAttribute, theme || 'light');
```

This could be simplified and made to support future themes. The default is the light theme; otherwise, use the specified theme.

Reply:
You are right. `cds-theme` attribute is `empty` for `light` and have value for `dark` theme only.  
Corrected in commit 12a2875.